app/views/favorites/index.html.erb

```diff
@@ -43,4 +43,17 @@
   </section>
   <% end %>
 
+  <section id="approved-pets" class="pet-list">
+    <% if !@pets.approved_pets.empty? %>
+    <h2 class="header-right">Approved Pets</h2>
+      <%  @pets.approved_pets.each do |pet| %>
```

Comment:
place links for pets that have approved applications. see pet-models for method call

Reply:
![](https://media.giphy.com/media/t9iEBuwqEpiGk/giphy.gif)